met-web/src/components/publicDashboard/ErrorBox.tsx

```diff
@@ -33,13 +36,20 @@ export const ErrorBox = ({
                 textAlign={'center'}
                 height="100%"
             >
-                <Grid item>
-                    <MetHeader4 color="error">Error</MetHeader4>
-                    <MetBody color="error">Click to reload</MetBody>
-                    <IconButton onClick={() => onClick()}>
-                        <LoopIcon sx={{ color: DASHBOARD.KPI.RADIALBAR.FILL_COLOR, fontSize: '4em' }} />
-                    </IconButton>
-                </Grid>
+                <When condition={!noData}>
```

Comment:
can you use \<I\f> \<Else\> and \<Then\> instead of \<When\> for this this please